test/test-pages/002/expected-metadata.json

```diff
@@ -0,0 +1 @@
+{"title":"This API is so Fetching! ✩ Mozilla Hacks – the Web developer blog","byline":"blog.nikhilism.com","excerpt":"For more than a decade the Web has used XMLHttpRequest (XHR) to achieve asynchronous requests in JavaScript. While very useful, XHR is not a very ..."}
```

Comment:
Probably not worth the effort, but these may be more human readable if we pretty-fy them (ie: `JSON.stringify(data, null, 2)`).